index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<html>
+<head>
+    <title>Sobia Furqan's Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet"
+          href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+<nav>
+    <section>
+        <div class="nav-links">
+            <a href="#home" class="home">Home</a>
+            <a href="#about-me" class="about-me">About Me</a>
+            <a href="#projects" class="work">Projects</a>
+            <a href="#contact" class="contact">Contact</a>
+    </section>
+    </div>
+</nav>
+<div id="home" class="hero-image">
+    <div class="hero-text">
+        <h1>Sobia Furqan</h1>
+        <p>Software Developer</p>
+        <button> <a href="#contact">Contact</a></button>
+    </div>
+</div>
+<section id="about-me">
+    <h2>About Me</h2>
+    <p>I am a software developer with a deep passion for understanding how technology can help people and enable me to create amazing apps as well as software. I'm currently training at Mayden Academy while learning the following languages: HTML, CSS, Js and PHP. I'm on week 2 and can't wait to learn more.</p>
+    <p>When not coding my hobbies include sports such as basketball, bouldering and longboarding, photography, travelling, learning languages (mainly Korean) and exploring museums, cafes / restaurants, exhibitions.</p>
+</section>
+<main>
+    <section>
+        <h2>Projects</h2>
+        <section>
```

Comment:
Does this section need to be here? it looks like it's unclosed and I think it can just be removed without causing any issues!

Reply:
unfortunately needs to be there for formatting purposes. Didn't notice the closing was unclosed, many thanks!